doc/api/stream.md

```diff
@@ -1095,6 +1095,9 @@ constructor and implement the `writable._write()` method. The
   * `decodeStrings` {Boolean} Whether or not to decode strings into
     Buffers before passing them to [`stream._write()`][stream-_write].
     Defaults to `true`
+  * `decodeBuffers` {Boolean} Whether or not to decode Buffers into strings
+    using the default encoding before passing them to
+    [`stream._write()`][stream-_write].
```

Comment:
It might be beneficial to document the behavior if both `decodeStrings` and `decodeBuffers` are set to `true`.